src/index.js

```diff
@@ -1,11 +1,23 @@
 var bodyParser = require('body-parser');
 var app = require('express')();
 var handleWfhRequest = require('./handle-wfh-request');
+var parseText = require('./parse-text')
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.listen(process.env.PORT);
 
 app.post('/wfh', (req, res) => {
 	res.send();
-	handleWfhRequest(req.body.user_id, req.body.response_url);
+	if (parseText.checkIfClear(req.body.text))
+	{
+		handleWfhRequest.clear(req.body.user_id, req.body.response_url, parseText.getDate(req.body.text));
+	}
+	else if (parseText.checkIfDateTimeInterval(req.body.text))
```

Comment:
This check could go into the `handleRequest()` method and then the actual google calendar call doesn't have to be known by the caller like I mentioned before.